src/elements/TabbedContainer/TabbedContainer.js

```diff
@@ -11,76 +11,44 @@ class TabbedContainer extends Component {
 	constructor(props) {
 		super(props);
 
+		const defaultActiveIndex = Math.max(React.Children.toArray(this.props.children).findIndex(c => c.props.defaultActive), 0);
+
 		this.state = {
-			// set default showing tab (0 = first tab, 1 = second tab, etc...)
-			activeIndex: 0
+			activeIndex: defaultActiveIndex
 		};
 
 		this.changeTabs = this.changeTabs.bind(this);
-		this.renderNavItems = this.renderNavItems.bind(this);
-		this.renderOneNavItem = this.renderOneNavItem.bind(this);
 	}
 
-	changeTabs(event, key) {
-		event.preventDefault();
+	changeTabs(e) {
+		e.preventDefault();
 
 		this.setState({
-			activeIndex: key
+			activeIndex: Number(e.target.dataset.tab)
 		});
 	}
 
-	renderNavItems(key) {
-		const tab = this.props.children[key] ? this.props.children[key] : null,
-			keyNum = Number(key);
-
-		const navItemClasses = classNames(
-			this.state.activeIndex === keyNum ? 'active' : ''
-		);
-
-		return (
-			<li key={key} className={navItemClasses}>
-				<a href="#" onClick={ (e) => this.changeTabs(e, keyNum) }>{ tab.props.title }</a>
-			</li>
-		);
-	}
-
-	renderOneNavItem() {
-		const oneTab = React.Children.only(this.props.children);
-
-		return (
-			<li className="active">
-				<a href="#">{ oneTab.props.title }</a>
-			</li>
-		);
-	}
-
 	render() {
-		let index = 0,
-			renderNav;
-		const isActive = this.state.activeIndex;
+		const { activeIndex } = this.state,
+			{ children } = this.props;
 
-		const tabs = React.Children.map(this.props.children, (child) => {
+		const tabs = React.Children.map(children, (child, i) => {
 			return React.cloneElement(child, {
-				active: child.props.active === true ? true : (isActive === index++)
+				active: child.props.active || activeIndex === i
```

Comment:
This you can now remove the check for `active` in child props, since that won't be passed ever